src/v2/view-builder/views/PollView.js

```diff
@@ -44,7 +45,13 @@ const Body = BaseForm.extend(Object.assign(
     remove () {
       BaseForm.prototype.remove.apply(this, arguments);
       this.stopPolling();
-    }
+    },
+
+    triggerAfterError (model, error) {
+      this.stopPolling();
```

Comment:
are you able to do
```
super.triggerAfterError()
this.stopPolling();
this.model.trigger('clearFormError');
```

Reply:
Can not use `clearFormError ` because need is to remove form content (that shows polling in progress) and just show the error..

Added a new handler instead. Thanks for the pointer, its much better now.

Reply:
Hide form content on error seems very specific to this view hence don't add event/handler to BaseForm but keep in this PollView?
Sorry for missing detail earlier.

Reply:
It would be helpful, there are many instances where devices also does the polling so having a generalized method will help, i think it's a good addition to keep it in baseForm.

Reply:
Oh, interesting, would that make sense to have a Base View for Polling to abstract the common logic?

Reply:
polling.js shares the basic workings of polling functions.. No harm creating a new BasePolling view, thats a good idea. Device's team use cases are complex for eg. `DeviceChallengePollView`

Reply:
In my opinion start and stop poll controls should be handled by the view and not by the baseview. I don't see there is much common logic left then.

Reply:
> there are many instances where devices also does the polling so having a generalized method will help, i think it's a good addition to keep it in baseForm.

My point is the removing polling view from Form is specific to Polling related views but not BaseForm hence worth to consider not to put into BaseForm but use other fashions.

Reply:
oh.. why i added to baseForm is it will be useful in future for the other views if they want to clear out form fields, like we clear out the form errors. I am also planning to add handler that can clear form errors instead of using `this.$el.find('.o-form-error-container').empty();` explicitly..

Reply:
iirc, BaseForm.clearError will clear all errors.

Reply:
we have an handler for that in BaseForm, but at some places devs don't used it. ```  handleClearFormError () {
    if (this.$('.o-form-error-container').hasClass('o-form-has-errors')) {
      this.clearErrors();
    }
  }```

Reply:
yeah, that's what I meant. It meant for people to use to clear errors. (could be because we introduce this event later and didn't find time to refactor all places to avoid redundancy.)